### ISSUE TITLE
Remote dashboard creation 401 Unauthorized error

### ISSUE THREAD

**Issue by Booma-Baskaran** [Hygieia/Hygieia#2853]
Hi,

I am trying to automate the dashboard creation using Hygieia API by making http://localhost:8080/api/dashboard/remoteCreate POST call. Below are the responses from swagger and Postman,

Swagger:
![image](https://user-images.githubusercontent.com/36442233/56682993-4e835900-6692-11e9-826d-d1cba80db044.png)

Postman:
I have generated the API Token using Admin menu and then did base64 encoding though I am getting 401 error,

![image](https://user-images.githubusercontent.com/36442233/56683388-09abf200-6693-11e9-9cbd-108fd41872ba.png)

Thanks,
Booma Baskaran.

**Comment by rvema** [comment]
Can you check if the admin token was created in the Mongo DB?

**Comment by Booma-Baskaran** [comment]
Yes. 'apitoken' collection has the apiUser, apiKey and expirationDt.

**Comment by Sbrenthughes** [comment]
what command did you use to encode the token?

**Comment by Booma-Baskaran** [comment]
Nope. Using Base64 Encode in the Notepad++ by going to the Plugin commands option since I use Windows10.

![ApiToken-Base64Encoding](https://user-images.githubusercontent.com/36442233/56759962-c23f6780-675f-11e9-9eeb-1d731e4382c8.jpg)

**Comment by Sbrenthughes** [comment]
Can you try using `PasswordIsAuthToken:{\"apiKey\":\"<your_token>\"}` when you do this?

**Comment by Booma-Baskaran** [comment]
Yes. I have done the same. I already have generated the apiToken by encoding PasswordIsAuthToken:{"apiKey":"<your_token>"} but still no luck. Is there any way to ignore apiToken?

**Comment by Sbrenthughes** [comment]
I am not sure why you want wan't insecure apis manipulating data? You could also try doing the java route for encoding https://hygieia.github.io/Hygieia/api.html#basic-authentication-for-secure-apis

**Comment by Sbrenthughes** [comment]
@Booma-Baskaran Can you reach out to me on https://gitter.im/capitalone/Hygieia. I can work with you directly.

**Comment by Sbrenthughes** [comment]
@Booma-Baskaran any update here?

**Comment by Booma-Baskaran** [comment]
@Sbrenthughes Bit busy on other priority items. Are you available by next week? What would be the good time to ping you in gitter?

**Comment by Sbrenthughes** [comment]
Reach out anytime

**Comment by Booma-Baskaran** [comment]
@Sbrenthughes I fixed this by creating a simple java program with the steps mentioned in https://hygieia.github.io/Hygieia/api.html (Basic Authentication for Secure APIs section). After that, content-type with application/json header missing in the Postman call. Fixed it.

Now, I can able to make the Rest calls but I am not sure about the required input parameters. Can you please post here if you have any sample request parameters?

**Comment by Sbrenthughes** [comment]
```
{
    "codeRepoEntries": [
        {
            "toolName": "GitHub",
            "description": "description.",
            "options": {
                "branch": "master",
                "url": "https://github.com/hygieia/hygieia-core"
            }
        },
        {
            "toolName": "GitHub",
            "description": "description.",
            "options": {
                "branch": "master",
                "url": "https://github.com/hygieia/hygieia"
            }
        }
    ],
    "metaData": {
        "template": "CapOne",
        "title": "testTitle",
        "type": "Team",
        "applicationName": "testAPP",
        "componentName": "testAPP",
        "businessService": "",
        "businessApplication": "",
        "owner": {
            "username": "ownerTest",
            "authType": "STANDARD"
        }
    }
}
```
